templates/next/pages/foos/[id]/index.tsx

```diff
@@ -25,7 +25,7 @@ const Page: NextComponentType<NextPageContext, Props, Props> = (props) => {
           <title>{`Show {{{ucf}}} ${ {{~lc}}['@id'] }`}</title>
         </Head>
       </div>
-      <Show {{{lc}}}={ {{{lc}}} } />
+      <Show {{{lc}}}={ {{{lc}}} text=text} />
```

Comment:
Maybe?

```suggestion
      <Show {{{lc}}}={ {{{lc}}} } text=text />
```

(I've not tested it)